<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Text_Dialog</class>
 <widget class="QDialog" name="Text_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>388</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>388</width>
    <height>485</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Text Options</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>11</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text font</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>47</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>74</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>102</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>74</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vert just</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vert spacing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>47</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Horz just</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Halo colour</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>180</y>
     <width>357</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Style</string>
   </property>
   <widget class="QCheckBox" name="_107">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>25</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_109">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_115">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>71</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Halo</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_108">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>25</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_110">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Boxed</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_111">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>25</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Outline</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="_112">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>48</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Scaled</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="_117">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>132</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Text height in points</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>448</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_114">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>448</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_118">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>448</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clone-All</string>
   </property>
  </widget>
  <widget class="QComboBox" name="_104">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>46</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="_105">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>72</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>448</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="_113">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>290</y>
     <width>307</width>
     <height>151</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="interactive">
    <bool>false</bool>
   </property>
   <property name="sceneRect">
    <rectf>
     <x>0.000000000000000</x>
     <y>0.000000000000000</y>
     <width>300.000000000000000</width>
     <height>150.000000000000000</height>
    </rectf>
   </property>
   <property name="resizeAnchor">
    <enum>QGraphicsView::NoAnchor</enum>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_101">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>48</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_102">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>74</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_103">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>102</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>-360.000000000000000</double>
   </property>
   <property name="maximum">
    <double>360.000000000000000</double>
   </property>
  </widget>
  <widget class="RCDoubleSpinBox" name="_106">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>100</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>-100000000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="_116">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>158</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QFontComboBox" name="_100">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>10</y>
     <width>255</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="fontFilters">
    <set>QFontComboBox::ProportionalFonts|QFontComboBox::ScalableFonts</set>
   </property>
  </widget>
  <widget class="QPushButton" name="_2000">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>157</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select Colour</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../RealCAD.qrc">
     <normaloff>:/images/PROPERTIES-colours.svg</normaloff>:/images/PROPERTIES-colours.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="_1000">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>22</y>
     <width>255</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="_1001">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>52</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1002">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>78</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1003">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>106</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1116">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>162</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1006">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>104</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1005">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>76</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1004">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>50</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="_1011">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>14</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Default</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/undo.svg</normaloff>:/images/undo.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>11</width>
     <height>11</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_14</zorder>
  <zorder>_117</zorder>
  <zorder>_1</zorder>
  <zorder>_114</zorder>
  <zorder>_118</zorder>
  <zorder>_104</zorder>
  <zorder>_105</zorder>
  <zorder>_2</zorder>
  <zorder>_113</zorder>
  <zorder>_101</zorder>
  <zorder>_102</zorder>
  <zorder>_103</zorder>
  <zorder>_106</zorder>
  <zorder>_116</zorder>
  <zorder>_2000</zorder>
  <zorder>_1000</zorder>
  <zorder>_100</zorder>
  <zorder>_1001</zorder>
  <zorder>_1002</zorder>
  <zorder>_1003</zorder>
  <zorder>_1116</zorder>
  <zorder>_1006</zorder>
  <zorder>_1005</zorder>
  <zorder>_1004</zorder>
  <zorder>_1011</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RCDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../RealCADQt/source/rcdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_100</tabstop>
  <tabstop>_101</tabstop>
  <tabstop>_102</tabstop>
  <tabstop>_103</tabstop>
  <tabstop>_104</tabstop>
  <tabstop>_105</tabstop>
  <tabstop>_106</tabstop>
  <tabstop>_116</tabstop>
  <tabstop>_2000</tabstop>
  <tabstop>_117</tabstop>
  <tabstop>_107</tabstop>
  <tabstop>_115</tabstop>
  <tabstop>_108</tabstop>
  <tabstop>_110</tabstop>
  <tabstop>_111</tabstop>
  <tabstop>_112</tabstop>
  <tabstop>_1</tabstop>
  <tabstop>_114</tabstop>
  <tabstop>_118</tabstop>
  <tabstop>_2</tabstop>
  <tabstop>_113</tabstop>
  <tabstop>_109</tabstop>
 </tabstops>
 <resources>
  <include location="../RealCAD.qrc"/>
 </resources>
 <connections/>
</ui>
